app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,7 @@
+class IngredientsController < ApplicationController
+  
+  def index
```

Comment:
This controller design is okay for the US, but another good choice could be nesting the ingredients_controller under `chefs`. So, you have a directory under controllers called `chefs` and then a nested ingredients_controller in there. This helps indicate that this list of ingredients is for a specific chef